File: 2/PHP/Product.php
<?php

class Product{
	// Atribut Private
	private $idProduct;
	private $price;

	// Konstruktor
	function __construct() {
		$idProduct = "";
		$price = 0;
	}

	// Set & Get

	// ID Product
	function setIDProduct($idProduct) {
		$this->idProduct = $idProduct;
	}
	function getIDProduct() {
		return $this->idProduct;
	}

	// Price
	function setPrice($price) {
		$this->price = $price;
	}
	function getPrice() {
		return $this->price;
	}
}

?>

File: 2/Python/Product.py
class Product:
	# Konstruktor
	def __init__(self):
		self.__idProduct = ""
		self.__price = 0

	# Set & Get

	# ID Product
	def setIDProduct(self, idProduct):
		self.__idProduct = idProduct

	def getIDProduct(self):
		return self.__idProduct

	# Price
	def setPrice(self, price):
		self.__price = price

	def getPrice(self):
		return self.__price